ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -264,6 +306,19 @@ open class CommandBar: UIView {
         containerMaskLayer.locations = locations.map { NSNumber(value: Float($0)) }
     }
 
+    private func updateButtonTokens() {
+        leadingCommandGroupsView.updateButtonGroupViews()
+        mainCommandGroupsView.updateButtonGroupViews()
+        trailingCommandGroupsView.updateButtonGroupViews()
+    }
+
+    @objc private func handleCommandButtonTapped(_ sender: CommandBarButton) {
```

Comment:
Is this method needed? I believe this is also in `CommandBarCommandGroupsView`.

Reply:
Looks like it's being used in the groups view but not here, as it looks like it was deleted in #987. I can go ahead and remove this method